resources/main.js

```diff
@@ -4,4 +4,11 @@ $(document).ready(function () {
         edge: 'right',
         closeOnClick: true
     });
+
+    $('#analytics-link').on('click', function() {
```

Comment:
Maybe move this block to a new function, for greater testability, and then add a specific test.